Extension/Javascript-Extension/Visual-Studio-Code/create-project.md

```diff
@@ -17,11 +17,11 @@ The following steps help you create **Syncfusion Web Applications** through the
 
 1. In Visual Studio Code, open the command palette by pressing **Ctrl+Shift+P**. The Visual Studio Code palette opens, search the word **Syncfusion**, so you can get the templates provided.
 
-    ![CreateProjectPalette](images/CreateProjectPalette.png)
+    ![Create Project Palette in JavaScript extension](create-project_images/javascript-extension-project-palette.png)
```

Comment:
Image Folder name is wrong, check my comments in this PR.
https://github.com/syncfusion-content/extension-docs/pull/94

Reply:
resolved